tests/model_registry/test_model_registry_scc.py

```diff
@@ -0,0 +1,149 @@
+import pytest
+from typing import Self
```

Comment:
if we do not reach an agreement on bumping the python version we should use 
```
from __future__ import annotations 
from typing_extensions import Self
```

Reply:
`from __future__ import annotations ` is not needed for 3.11 onwards, from what I remember. But I will add it, till version is bumped.